packages/blockchain-wallet-v4-frontend/src/data/auth/sagaRegister.ts

```diff
@@ -15,6 +15,6 @@ export default ({ api, coreSagas, networks }) => {
     yield takeLatest(actions.triggerWalletMagicLink.type, authSagas.triggerWalletMagicLink)
     yield takeLatest(actions.exchangeLogin.type, authSagas.exchangeLogin)
     yield takeLatest(actions.exchangeResetPassword.type, authSagas.exchangeResetPassword)
-    yield takeLatest(actions.continueLoginProcess, authSagas.continueLoginProcess)
+    yield takeLatest(actions.continueLoginProcess.type, authSagas.continueLoginProcess)
```

Comment:
Which is the idea of using `.type`? 
Not a review comment just a question to understand that convention

Reply:
when we added RTK I believe we use type from that time, and I notice on some places in recent code it has been skipped, so I just used it to be on same line. We might discuss it as team and make some coding standard for usage of type